### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by VaibhavWakde52**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ along with other very useful endpoints.
 - NPM for managing dependencies.
 - Fastify as the web framework
 - Apollo for GraphQL
-- Typeorm as a database layer
+- Typeform as a database layer
```
Comment (rebelchris) [inline README.md:29]: Its actually Typeorm 👍

**Review by rebelchris — CHANGES_REQUESTED** [review]
Thanks for the fixes, just one note on a wrong change.

File: README.md (hunk 1 above)
Comment (VaibhavWakde52) [reply README.md:29]: ohhh ,my bad , i will change it back

**Review by rebelchris — APPROVED** [review]
Thank you so much

**Review by idoshamun — APPROVED** [review]
Thank you very much!

**Pull request merged**